web/pages/semantics.md

```diff
@@ -0,0 +1,64 @@
+# Semantics in K
+
```

Comment:
```suggestion
# Projects using K

```

Reply:
@shd101wyy please also rename this file to `projects.md`, and the sidebar link to say "Projects using K"